exercise3-1/src/camera_demo/src/application_nodes.cc

```diff
@@ -51,8 +51,8 @@ void CameraProcessingNode::DataLoggerCallback(const FakeImage::SharedPtr image)
   {
     auto span = tracer_data_logger_->WithSpan("DataLogger");
 
-    // Assume it takes 1ms to serialize the data which is all on the CPU
-    WasteTime(std::chrono::microseconds(1000));
+    // Assume it takes 6ms to serialize the data which is all on the CPU
+    WasteTime(std::chrono::microseconds(6000));
 
     // Assume it takes about 1ms to write the data where it is blocking but yielded to the CPU.
     std::this_thread::sleep_for(std::chrono::microseconds(1000));
```

Comment:
You missed adjusting this. The original intention is to simulate 1ms of CPU processing time and 1ms of sleep (io wait for example). Now it's 6ms of CPU processing and 1 ms of sleep. This is fine I suppose.